docs/modules/plugins/pages/plugin-mobile-app.adoc

```diff
@@ -189,6 +201,20 @@ selenium.grid.platform-name=Android
 selenium.grid.automation-name=UIAutomator2
 ----
 
+=== Local
+:tlocal-profile: mobile_app/local
+
+The profile simplifies execution tests against the local appium server. It could be used in conjunction with: `{android-profile}`, `{ios-profile}`, `{tvOS-profile}`
```

Comment:
```suggestion
The profile simplifies execution of tests against the local Appium server. It could be used in conjunction with: `{android-profile}`, `{ios-profile}`, `{tvos-profile}`
```